doc/source/data/api/grouped_data.rst

```diff
@@ -8,44 +8,5 @@ GroupedData API
 GroupedData objects are returned by groupby call: 
 :meth:`Dataset.groupby() <ray.data.Dataset.groupby>`.
 
-Constructor
------------
-
-.. autosummary::
-   :nosignatures:
-   :toctree: doc/
-
-   grouped_data.GroupedData
-
-Computations / Descriptive Stats
---------------------------------
-
-.. autosummary::
-   :nosignatures:
-   :toctree: doc/
-
-   grouped_data.GroupedData.count
-   grouped_data.GroupedData.sum
-   grouped_data.GroupedData.min
-   grouped_data.GroupedData.max
-   grouped_data.GroupedData.mean
-   grouped_data.GroupedData.std
-
-Function Application
---------------------
-
-.. autosummary::
-   :nosignatures:
-   :toctree: doc/
-
-   grouped_data.GroupedData.aggregate
-   grouped_data.GroupedData.map_groups
-
-Aggregate Function
-------------------
-
-.. autosummary::
-   :nosignatures:
-   :toctree: doc/
-
-   aggregate.AggregateFn
+.. include:: ray.data.grouped_data.GroupedData.rst
```

Comment:
@can-anyscale  not blocking since this is already an issue, but is there a way to hide the constructor for `GroupedData`? It shouldn't be exposed to the user

Reply:
lol i see, i think so yes - but without a constructor, how users can create an object to invoke the its public methods?

Reply:
Users should call the `Dataset.groupby()` method

Reply:
got you